### ISSUE TITLE
Mappers should be warned about the risks of accepting MAPS.ME edits as fact

### ISSUE THREAD

**Issue by SomeoneElseOSM** [maproulette/maproulette2#348]
See http://www.openstreetmap.org/node/4697076691/history and http://www.openstreetmap.org/changeset/52817169 .  Here a MAPS.ME-added "embassy" had some extra tags added to it by a well-meaning mapper, but unfortunately "embassy" is one of those tags that MAPS.ME users tend to use for personal markers.  In this case basic QA was not done and the embassy was "made to look legitimate" by a Maproulette edit.

A solution here could either be within Maproulette itself (e.g. warning of the dangers of editing in areas with which the mapper is not familiar) or on a challenge by challenge basis, or both.

**Comment by mvexel** [comment]
Yep, I am taking this as a priority item to add to the upcoming version.

**Comment by mvexel** [comment]
How about this wording the first time a user starts to edit something:

"MapRoulette allows you to fix problems in OSM anywhere in the world. Please take extra care when editing things in OSM in areas that you don't know. If you are not sure what to do, it is always best to skip the task and leave it to someone else."

**Comment by SomeoneElseOSM** [comment]
@mvexel Yes - that sounds OK.  I might have said "you aren't familiar with" instead of "you don't know", but "don't know" is probably better for people without English as a first language.
